src/assets/i18n/fr.json

```diff
@@ -29,6 +29,7 @@
 	"Message": "Message",
 	"Module": "Module",
 	"Nameservers": "Serveurs DNS",
+	"NS required": "NS obligatoire",
```

Comment:
There are some other new translatable strings (e.g. 'Digest type required', 'Algorithm required', ...). Do you plan on providing a translation within this PR? Or should we do it in other PRs?

Reply:
I've updated the translation.